dashboard/app/models/user.rb

```diff
@@ -328,6 +328,7 @@ def district_teachers(cohort = nil)
     ['gender.none', '-']
   ].freeze
 
+  attr_accessor :ip
```

Comment:
If necessary, this should be (well) documented. Is there a reason not to use `User.current_sign_in_ip`?